jaggr-core/WebContent/dojo/_loaderExt.js

```diff
@@ -280,6 +288,22 @@ combo.isI18nResource = combo.isI18nResource || function(mid) {
 	}
 };
 
+combo.addBootLayerDeps = function(deps) {
+	excludes = excludes.concat(deps);
+};
+
+/*
+ * Returns true if the specified module is defined, false otherwise
+ */
+combo.isDefined = function(name) {
+	try {
+		require(name);
+		return true;
+	} catch (ignore) {
+	}
+	return false;

```

Comment:
Not really any different, but it may make sense stylistically to move `return false;` into the catch block.